todoapp/src/App.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Todo from './Todo';
+import todos from './todo.json';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+     this.state = {
+      todos: [],
+      newToDoItem: {
+      text: ''        
+      }
+    };
+
+    this.changeToDoList = this.changeToDoList.bind(this);
+    this.inputAdd = this.inputAdd.bind(this);
+    this.submitNewItem = this.submitNewItem.bind(this);
+  }
+  componentDidMount() {
+      setTimeout(() => {
+      this.setState({ todos });
+    }, 1000);
+  }
+
+  changeToDoList(updatedReadValue, toDoId) {
+    const updatedtodos = this.state.todos.map(comment => {
+      if (toDoId === comment.id) {
+        return Object.assign({}, comment, {read: updatedReadValue});
+      }
+      return comment;
+    });
+    this.setState({
+      todos: updatedtodos
+    });
+  }
+
+  rendertodos(todos) {
```

Comment:
`rendertodos`
**camelCase 😬**

the other thing is not required but I think it's better to have a separated file for a new component you are creating.

it's good use of a very tiny change in a methods, but if you used to have more than a one file you will have better understanding of each component on it's own.

you can pass any needed things here to it by `props` and it will work exactly the same.

so you don't have to import all your components in the App component file but making them on a multiple files.